scripts/ci/automation_full_test.py

```diff
@@ -579,12 +580,16 @@ def run_instance_modules(self, instance_modules):
             azdev_test_result_fp = os.path.join(azdev_test_result_dir, f"test_results_{python_version}_{profile}_{instance_idx}.parallel.xml")
             cmd = ['azdev', 'test', '--no-exitfirst', '--verbose'] + parallel_tests + \
                   ['--profile', f'{profile}', '--xml-path', azdev_test_result_fp, '--pytest-args', '-o junit_family=xunit1 --durations=10 --tb=no']
+            if enable_traceback:
+                cmd[-1] = '-o junit_family=xunit1 --durations=10'
             parallel_error_flag = process_test(cmd, azdev_test_result_fp, live_rerun=fix_failure_tests)
             pipeline_result = get_pipeline_result(azdev_test_result_fp, pipeline_result) if enable_pipeline_result else None
         if serial_tests:
             azdev_test_result_fp = os.path.join(azdev_test_result_dir, f"test_results_{python_version}_{profile}_{instance_idx}.serial.xml")
             cmd = ['azdev', 'test', '--no-exitfirst', '--verbose', '--series'] + serial_tests + \
                   ['--profile', f'{profile}', '--xml-path', azdev_test_result_fp, '--pytest-args', '-o junit_family=xunit1 --durations=10 --tb=no']
+            if enable_traceback:
+                cmd[-1] = '-o junit_family=xunit1 --durations=10'
```

Comment:
Shall we made `--tb=no/auto` be an extra variable in command?  So we don't need to duplicate these two lines.

Reply:
Something like this?
```
pytest_args = '-o junit_family=xunit1 --durations=10'
if enable_traceback:
    pytest_args += ' --tb=no'
cmd = ['azdev', 'test', '--no-exitfirst', '--verbose', '--series'] + serial_tests + \
      ['--profile', f'{profile}', '--xml-path', azdev_test_result_fp, '--pytest-args', pytest_args]
```

Reply:
Yes